pandas/core/series.py

```diff
@@ -190,6 +190,23 @@ class Series(base.IndexOpsMixin, generic.NDFrame):
         The name to give to the Series.
     copy : bool, default False
         Copy input data.
+
+    Examples
+    --------
+    Constructing Series from a dictionary with an Index specified
```

Comment:
hmm we don't have a good example. can you add the same, but use say index=['a', 'b']) to show what this does *before*

Reply:
Yes of course. Used the same values to show the difference